### ISSUE TITLE
MariaDB section README.md needs update

### ISSUE THREAD

**Issue by JragonWolf** [ethicalhack3r/DVWA#211]
you forgot the 

create user dvwa; 

command entirely in the README.md regarding how to set up MariaDB for proper usage, took me ( a noob) over a week to realize that. Please update.

**Comment by digininja** [comment]
The grant command should create the user for you, I'll check it to make
sure but it certainly used to.

**Comment by JragonWolf** [comment]
ya I thought so too, but after removing dvwa several times and going step by step, I had to manually add the user and it worked finally. could have been my system for whatever reason....Im a noob, so I honestly don't know, just knew it was a headache.

**Comment by digininja** [comment]
I'll give it a check and see.

**Comment by digininja** [comment]
I just tested this in Kali and it works fine for me. There is only the root user to begin with, I add the database and then do the grant then I log out and successfully log in as the dvwa user.

![image](https://user-images.githubusercontent.com/117081/34652901-697125be-f3dc-11e7-96a4-d5912be50244.png)

Looks like something went wrong at your end.

**Comment by haking4fun** [comment]
I just got DVWA setup on my raspberry pi using ubuntu 16.04. To get the database to connect I kept the dvwa database in place, and just created a user with a blank password. Here are my commands.

`sudo mysql -u root -p`
`create user dvwauser;`
`set assword for dvwauser =password("")'
`grant all PRIVILEGES on dvwa.* to dvwauser@localhost identified by '';`

Then make the changes to the config.inc.php file by updating the database variables.
db_server = 'localhost';
db_user = 'dvwauser';
db_password = '';
